### PR TITLE
sdks io for cassandra

### PR DESCRIPTION
**Pull request by surpass**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-SDKS-IO] cassandra io read and write
  

---

### CODE REVIEW

**Comment by jbonofre** [comment]
I guess you didn't see:

1. https://issues.apache.org/jira/browse/BEAM-245
2. https://github.com/apache/beam/pull/592

**Comment by jbonofre** [comment]
I would appreciate you close this PR and contribute directly on the existing one.

**Comment by jbonofre** [comment]
Thanks !

**Comment by davorbonaci** [comment]
First, @surpass, thank you for your contribution. It is much appreciated and we'd like to have Cassandra support in Beam as soon as possible!

Indeded, @jbonofre has started this particular effort a while ago, and it would be great if you can work together to move this IO forward as quickly as possible.

Personally, I don't think we need to worry which PR goes first. Perhaps both can go in, in some order, to get a few distinct benefits from each of them.

I'd be happy to assist, as appropriate. Also, CC: @jkff, @dhalperi.

**Comment by jbonofre** [comment]
Original Cassandra PR is almost ready. I would like to merge pretty fast.

File: sdks/java/io/cassandra/pom.xml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	</parent>
+
+	<artifactId>beam-sdks-java-io-cassandra</artifactId>
+	<name>Apache Beam :: SDKs :: Java :: IO :: Cassandra</name>
+	<description>IO to read and write on Cassandra datasource.</description>
+
+	<properties>
+		<cassandra-driver.version>3.1.4</cassandra-driver.version>
+		<netty.version>4.0.30.Final</netty.version>
+		<guava.version>16.0.1</guava.version>
```
Comment (XuMingmin) [inline sdks/java/io/cassandra/pom.xml]: This version is conflicted with default version 20.0, it may breaks.

**Comment by jbonofre** [comment]
As said, I'm resuming/finishing work on #592

File: sdks/java/io/cassandra/pom.xml (hunk 1 above)
Comment (zznate) [inline sdks/java/io/cassandra/pom.xml]: FYI this *just* changed in the Java-Driver to support v20: https://github.com/datastax/java-driver/commit/1ddd09eeb92bb381223e081fd691bc93c3ffa4fc and https://datastax-oss.atlassian.net/browse/JAVA-1328

Comment (zznate) [inline sdks/java/io/cassandra/pom.xml]: Netty rev as well: https://github.com/datastax/java-driver/blob/3.1.x/pom.xml#L54

**Comment by jbonofre** [comment]
I just take a quick look to compare with what I did in #592. IMHO, this one is not what we want in Beam: this IO is basically a copy of `JdbcIO` updated for Cassandra. So basically, no splitting, no leveraging Cassandra features. I would recommend to close this PR and take a look on mine (I'm finalizing #592 this week).

**Comment by davorbonaci** [comment]
CassandraIO has been merged to Beam from a separate pull request. This pull request is pending closure. Any objections?

**Comment by jbonofre** [comment]
+1 to close this PR. I asked already some weeks ago.

**Comment by jkff** [comment]
@surpass Could you please close this PR?

Asking in part because I use Github's "Mentioned" PRs tab to keep track of what I need to review, and there is no way for me to remove the PR from there except by asking you to close it.

**Pull request closed without merging**